editor/editor_themes.cpp

```diff
@@ -724,14 +724,56 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_icon("visibility_xray", "PopupMenu", theme->get_icon("GuiVisibilityXray", "EditorIcons"));
 	theme->set_constant("vseparation", "PopupMenu", (extra_spacing + default_margin_size + 1) * EDSCALE);
 
-	Ref<StyleBoxFlat> sub_inspector_bg = make_flat_stylebox(dark_color_1.lerp(accent_color, 0.08), 2, 0, 2, 2);
-	sub_inspector_bg->set_border_width(SIDE_LEFT, 2);
-	sub_inspector_bg->set_border_width(SIDE_RIGHT, 2);
-	sub_inspector_bg->set_border_width(SIDE_BOTTOM, 2);
-	sub_inspector_bg->set_border_color(accent_color * Color(1, 1, 1, 0.3));
-	sub_inspector_bg->set_draw_center(true);
-
-	theme->set_stylebox("sub_inspector_bg", "Editor", sub_inspector_bg);
+	for (int i = 0; i < 16; i++) {
```

Comment:
Do we even need to go to 16? If the impact on editor start performance is negligible it's fine, but otherwise I think 8 might be plenty enough, especially if it rotates.

Reply:
performance is not a problem, and it just ensures the rotation is slow and smooth in case it needs to be.